### ISSUE TITLE
Add Cisco virtual IOS to the test environment

### ISSUE THREAD

**Issue by danderson** [google/metallb#23]
Similar to #22, but for Cisco. The hard part with Cisco is that they do _not_ provide free trial versions of the virtual IOS appliance. You have to buy a license for $$$. Alternatively, [GNS3](https://www.gns3.com/) has an IOS emulator that can run normal, non-virtual IOS images inside an emulated router environment. To use that emulator, you just need to own _any_ IOS-using equipement, which allows you to download the IOS firmware images.

Bottom line, Cisco might be much harder/impossible to get working in a way that many people can test with, but we should explore it more completely and document how to do it, or document why it's not possible.
